svthalia/concrexit
# Add MoneybirdProject to fix pagination issues

This way we don't (fail to) check on a paginated API whether a project exists there already. Should solve a large fraction of moneybird synch failures.

Closes #3166.

### How to test
Steps to test the changes you made:
1. On staging, create a new event, register and TPay for it.
2. Check sentry and what shows up on moneybird.
